ppml/trusted-big-data-ml/python/docker-gramine/base/entrypoint.sh

```diff
@@ -80,29 +80,26 @@ fi
 
 # Attestation
 if [ -z "$ATTESTATION" ]; then
-    echo "[INFO] Attestation is disabled!"
-    ATTESTATION="false"
-elif [ "$ATTESTATION" = "true" ]; then
-  echo "[INFO] Attestation is enabled!"
-  # Build ATTESTATION_COMMAND
-  if [ -z "$ATTESTATION_URL" ]; then
-    echo "[ERROR] Attestation is enabled, but ATTESTATION_URL is empty!"
-    echo "[INFO] PPML Application Exit!"
-    exit 1
-  fi
-  if [ -z "$ATTESTATION_ID" ]; then
```

Comment:
Have you checked ATTESTATION_ID are changed to APP_ID? I modified the env variable names in `attestation.sh`.

Reply:
fixed